fixtures/constant-with-namespace.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace HELLO {
+
+    /**
+     * Does something really cool!
+     */
+    function world() {
+
+    }
+
+    \HELLO;
+}
+
+namespace {
```

Comment:
to be precise: its a syntax error to have "global" code mixed with namespaced code in a single file:

https://3v4l.org/2T5oq